Sources/Extensions/SwiftStdlib/IntExtensions.swift

```diff
@@ -61,7 +61,37 @@ public extension Int {
 		}
 		return String(format: "\(sign)%ikk", abs / 100000)
 	}
-	
+
+    /// SwifterSwift: Array of digits of integer value.
+    public var digits: [Int] {
+        guard self != 0 else { return [0] }
+        var digits = [Int]()
+        var number = self
+
+        while number != 0 {
+            let x = (number % 10).abs
```

Comment:
Instead of calculate the abs on each iteration, maybe is better to apply the abs on `var number = self.abs`. That way you just have to calculate it once :)

Reply:
Good one! Done 😊👍🏻